pkgs/by-name/by/byedpi/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "byedpi";
+  version = "0.12";
+
+  src = fetchFromGitHub {
+    owner = "hufrea";
+    repo = "byedpi";
+    rev = "refs/tags/v${finalAttrs.version}";
```

Comment:
I believe that you can shorten this to just `"v${finalAttrs.version}"`, i.e. no need for `refs/tags/`.

Reply:
I checked some PRs and packages - some uses "v{version}", some "refs/tags/v{version}". If the long version is used, then there are some reasons for this (i.e. explicitness)? I couldn't find any recommendations about it.

Reply:
To differentiate with branches. Although imo fetching a branch is a stupid idea